index.html

```diff
@@ -2374,21 +2394,6 @@ <h3>
 consumption rules</a> in the [[INFRA]] specification.
         </p>
 
-        <p>
-The value of the <code>@context</code> object member MUST be ignored as this is
-reserved for JSON-LD consumers.
```

Comment:
Please replace the text above with that approved in the resolution https://www.w3.org/2019/did-wg/Meetings/Minutes/2020-11-05-did#resolution2 : "Note that the value of the @context object member will be ignored, if present."

Reply:
This suggestion may address this concern: https://github.com/w3c/did-core/pull/454/#discussion_r525328136